docs/introduction.rst

```diff
@@ -50,18 +50,30 @@ Below is a depiction of how the system works in regards to Near-real-time (NRT)
 Build Server and Client
 ---------------------------
 
-In the home directory.
+In the home directory is a Dockerfile that will build a base image that can be used for the client and server.
+This Dockerfile is based off of a Java-14 image, installs the distribution via gradle, and can be built like this:
 
 .. code-block::
-  ./gradlew clean installDist test
-
-Note: This code has been tested on *Java14*.
+  shell% docker build --tag nrtsearch .
```

Comment:
Can we keep the instructions to build and run locally as well?

Reply:
@sarthakn7  yes, of course.  That was a mistake to remove that. sorry. I added it back in. Please re-review the doc again.  I also added the dockerized commands as subsubsections.